app/screens/planner/planner-screen.tsx

```diff
@@ -186,7 +186,13 @@ export const PlannerScreen = observer(function PlannerScreen({ navigation }: Pla
   )
 
   return (
-    <Screen style={ROOT}>
+    <Screen
+      style={ROOT}
+      preset="fixed"
+      statusBar="light-content"
```

Comment:
Can we remove it so the system will set it automatically?

Right now it'll be hidden on light mode